README.md

```diff
@@ -13,6 +13,7 @@ app.use(csp({
   // Specify directives as normal.
   directives: {
     defaultSrc: ["'self'", 'default.com'],
+    scriptSrcElem: ["'self'", "'unsafe-inline'"],
```

Comment:
Could you move this between `objectSrc` and `upgradeInsecureRequests`? I don't think it should be so high up in the documentation.

Reply:
I have moved the placement of this directive as requested